q2_assembly/quast/quast.py

```diff
@@ -338,10 +334,13 @@
     Returns:
         a Pandas dataframe with the tabular data.
     """
-    filename = "transposed_report.tsv"
-    filepath = os.path.join(results_dir, "combined_reference", filename)
+    subdir = os.path.join(results_dir, "combined_reference")
+    if os.path.isdir(subdir):
```

Comment:
Do we do this to keep the results in the combined reference folder returned by QUAST?

Reply:
I think it happened to me that this folder did not exist (when references were not fetched), in which case the transposed_report was written somewhere else and something failed after this step so I added this little check :)